src/main/java/ru/practicum/shareit/item/dto/ItemMapper.java

```diff
@@ -0,0 +1,35 @@
+package ru.practicum.shareit.item.dto;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+import ru.practicum.shareit.item.model.Item;
+import ru.practicum.shareit.request.model.ItemRequest;
+
+import java.util.List;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class ItemMapper {
+
+    public static Item toEntity(ItemDto dto) {
+        Item item = new Item();
+        item.setName(dto.getName());
+        item.setDescription(dto.getDescription());
+        item.setAvailable(dto.getAvailable());
+        if (dto.getRequest() != null)
+            item.setRequest(new ItemRequest(dto.getRequest().getId()));
+        return item;
+    }
+
+    public static ItemDto toDto(Item item) {
+        ItemDto dto = new ItemDto();
+        dto.setId(item.getId());
+        dto.setName(item.getName());
+        dto.setDescription(item.getDescription());
+        dto.setAvailable(item.getAvailable());
+        return dto;
+    }
+
+    public static List<ItemDto> toDto(List<Item> items) {
+        return items.stream().map(ItemMapper::toDto).toList();
```

Comment:
Стримы принято бить по строкам для лучшей читаемости кода:
```suggestion
        return items.stream()
            .map(ItemMapper::toDto)
            .toList();
```

Reply:
код не актуален в связи с подключением к проекту библиотеки Mapsruct